lib/addons/src/hooks/useDOMRect.ts

```diff
@@ -0,0 +1,61 @@
+import { useState, useEffect, RefObject } from 'react';
+import ResizeObserver from 'resize-observer-polyfill';
```

Comment:
Is the polyfill ALWAYS included?

Reply:
Do not understand, as in is it always needed?

This is what caniuse.com say - if we can live with this then I will remove the polyfill

![Screenshot 2021-06-21 174221](https://user-images.githubusercontent.com/6172338/122789985-27e3fb80-d2b8-11eb-90a9-ee4bb6665588.png)

Reply:
This might be a solution instead

https://www.npmjs.com/package/resize-observer

It also sports an optional install feature out of the box to overwrite window.ResizeObserver on demand